### PR TITLE
Refine README.md

### PR DESCRIPTION
**Pull request by davidshq**
## Description
Updates to several README.md files.

## Types of changes
- Grammar/spelling corrections.
- Removed excess words.
- Broke out installation section for wp-env in readme, added section on software requirements.

### CODE REVIEW

File: packages/env/README.md [hunk 1]
```diff
@@ -15,26 +15,32 @@ The local environment will be available at http://localhost:8888.
...
 $ npm -g i @wordpress/env
 ```
 
 You're now ready to use `wp-env`!
 
-### Starting the environment
+### Starting the Environment
 
-First, ensure that Docker is running. You can do this by clicking on the Docker icon in the system tray or menu bar.
+First, ensure that Docker is running. You can do this by clocking on the Docker icon in the system tray or menu bar.
```
Comment (Aurorum) [inline packages/env/README.md]: I think `clicking` is correct here?

Comment (davidshq) [reply packages/env/README.md]: Yes, clicking is correct, sorry.

**Comment by davidshq** [comment]
Believe this was a transient issue not related to this code, could someone with permissions rerun Travis CI on this?

File: README.md [hunk 2]
```diff
@@ -7,12 +7,12 @@
...
+- **Discuss:** Conversations and discussions take place in the <a href="https://wordpress.slack.com/messages/C02QB2JS7">`#core-editor` channel on the Making WordPress Slack</a>.
+- **Contribute:** Development of Gutenberg happens in this GitHub repo. Get started by <a href="https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md">reading the contributor guidelines</a>.
 - **Learn:** <a href="https://wordpress.org/gutenberg/">Discover more about the project on WordPress.org</a>.
 
-**Gutenberg is more than an editor.** While the project is currently focused on building the new editor for WordPress, it doesn't end there. This lays the groundwork for a new model for WordPress Core that will ultimately impact the entire publishing experience of the platform.
+**Gutenberg is more than an editor.** While the project is currently focused on building the new post editor for WordPress, it doesn't end there. Instead it lays the groundwork for a new model for WordPress Core that will ultimately impact the entire publishing experience.
```
Comment (talldan) [inline README.md]: I'd say the second sentence could be replaced completely. Phase 1 was building the new post editor. We're now in phase 2 and already aiming for loftier goals 😄 
```suggestion
**Gutenberg is more than an editor.** The project lays the groundwork for a new model for WordPress Core that will ultimately impact the entire publishing experience.
```

**Comment by talldan** [comment]
I re-ran the tests, so 🤞 they pass.

**Comment by michelleweber** [comment]
Copy changes look fine.

**Comment by gziolo** [comment]
This documentation changes look good. Before we land them, this PR needs to get updated to resolve merge conflicts.

**Pull request closed without merging**